fixtures/features/api-elements/param-with-enum-default.json

```diff
@@ -86,7 +86,9 @@
                           "meta": {},
                           "attributes": {
                             "default": [
-                              "A"
```

Comment:
Was this a bug in swagger adapter? Enum's default is an array in APIB refract, right?